Assets/Scripts/Gamestate/MatchController.cs

```diff
@@ -43,6 +46,11 @@ public class MatchController : MonoBehaviour
     [SerializeField]
     private int rewardBase = 1;
 
+    public Timer roundTimer;
+
+    [SerializeField]
+    private HUDGlobalController globalHud;
```

Comment:
GlobalHUDController and globalHUD?

Reply:
As we have have a HUDController, I wanted to build on that specific name for something functionally similar. But I don't really have a strong opinion regarding this, so I can change it before merge

Reply:
I think we should rename the HUDController to something more specific though, like PlayerHUDController or LocalHUDController